packages/ghost-cli/lib/migrations.js

```diff
@@ -34,4 +45,8 @@ module.exports = [{
     before: '1.14.1',
     title: 'Fix Sqlite DB path',
     task: makeSqliteAbsolute
+}, {
+    before: '1.17.4',
```

Comment:
bumping the version here so it runs on instances created with CLI 1.18.0 or earlier (this change will be released as v1.18.0)

```suggestion
    before: '1.18.1',
```